### PR TITLE
Add Alternative Name in CSR

### PR DESCRIPTION
**Pull request by jderusse**
to allow you to create certificate for multiple domains

### CODE REVIEW

File: src/Ssl/CSRGenerator.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+[ req_distinguished_name ]
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+subjectAltName = @req_subject_alt_name
+[ req_subject_alt_name ]
+%s
+EOL;
+            $sslConfigDomains = [];
+            foreach (array_values($subjectAltName) as $index => $domain) {
```
Comment (tgalopin) [inline src/Ssl/CSRGenerator.php:73]: I usually put spaces before and after foreach (same for try) unless the line before/after is a bracket. I'm not sure it's in any convention, but I would like to keep code style consistent :) .

**Comment by jderusse** [comment]
Test is red on HHVM because of the lib used in test to asserts that the CSR contains the expected X09 attribute.
I have to find an other way to read CSR file :/

**Comment by jderusse** [comment]
fixed

**Comment by tgalopin** [comment]
Thanks!

**Pull request merged**